src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
@bloodbare @vangheem FYI: we really need your opinion here if we want to keep the Guillotina API and plone.restapi in sync.